tests/installation/welcome.pm

```diff
@@ -34,7 +37,10 @@ sub run() {
             send_key_until_needlematch 'scc-invalid-url-deleted', 'backspace';
             type_string get_var('SCC_URL_VALID');
             wait_still_screen 2;
-            wait_screen_change { send_key 'alt-o' };    # OK
+            # Press Ok to confirm scc url
+            wait_screen_change { send_key 'alt-o' };
+            # See poo#19832, sometimes manage to match same tag twice and test fails
+            wait_still_screen 5;
```

Comment:
This might be useful before every assert_screen in this module as you might have similar issue after match_has_tag('inst-welcome-confirm-self-update-server')